uplink/Cargo.toml

```diff
@@ -8,6 +8,7 @@ license = "MPL-2.0"
 [dependencies]
 rkyv = { version = "0.7", default-features = false, features = ["size_32", "alloc"] }
 bytecheck = { version = "0.6", default-features = false }
+wee_alloc = "0.4.5"
```

Comment:
Both are fine, but we should decide if we want the usage of `wee_alloc` to be enforced in `uplink`, or if we want to leave it to contract writers. Keep in mind that there's an incentive to keep the bytecode small anyway.